### PR TITLE
Feature/math entry whitespacing

### PR DESCRIPTION
**Pull request by ssipasseuth**
Added new configuration option "authorize white space" to control the Mathquill option spaceBehavesLikeTab

### CODE REVIEW

File: views/js/pciCreator/dev/mathEntryInteraction/creator/tpl/propertiesForm.tpl [hunk 1]
```diff
@@ -11,6 +11,17 @@
 </div>
 
 <hr />
+<h3>{{__ "Special Settings"}}</h3>
+
+<div>
+    <label class="panel">
+        <input name="spaceBehavesLikeTab" type="checkbox" {{#if spaceBehavesLikeTab}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span>
+        {{__ "prevent white space"}}
```
Comment (no-chris) [inline views/js/pciCreator/dev/mathEntryInteraction/creator/tpl/propertiesForm.tpl]: Tom once made a very useful comment about options and UX, that we should always tick a checkbox to _activate_ or _enable_ something, not the other way around which confuses the poor user's brain ;-)

Comment (ssipasseuth) [inline views/js/pciCreator/dev/mathEntryInteraction/creator/tpl/propertiesForm.tpl]: Yes that is right. I confess I just wanted to make this quick.
Ok wait, I will change that.

**Comment by ssipasseuth** [comment]
ok, I changed **prevent** to **authorize**, please check again

File: views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js [hunk 2]
```diff
@@ -86,16 +87,25 @@ define([
...
+                    handlers: {
+                        edit: function() {
+                            self.trigger('responseChange');
+                        }
                     }
-                }
-            });
+                };
+
+            console.log(this.config);
```
Comment (no-chris) [inline views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js]: Forgot something ? ;)

**Pull request merged**